### PR TITLE
Fix logging

### LINKED ISSUE #1762: logging to /etc
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

_6_

---
**[BUG REPORT | OTHER]:**

Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[BUG | ISSUE] Expected Behaviour:**
Logs should be created in `/var/log`  (ref: http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/)

**[BUG | ISSUE] Actual Behaviour:**
Installer creates a log in `/etc/pihole`

**[BUG | ISSUE] Steps to reproduce:**
Install Pi-Hole and inspect output from `ls -al /etc/pihole`
Note the presence of the file `install.log`

If this was done intentionally I'm very curious to understand the reasoning behind that.  
If not, I could help by submitting a pull request to correct this. In the latter case, on which branch would you like me to base the changes?

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### PR DESCRIPTION
**Pull request by Mausy5043**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

_7_

---
Changed the location of the logfile (install.log) from `/etc` to `/var/log` as is appropriate.
This PR# fixes #1762.
Mentioning also PR #1699 because this issue is also discussed there. :wink:

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -2114,7 +2114,7 @@ main() {
   fi
 
   # Display where the log file is
-  echo -e "\\n  ${INFO} The install log is located at: /etc/pihole/install.log
+  echo -e "\\n  ${INFO} The install log is located at: /var/log/pihole-install.log
```
Comment (jacobsalmela) [inline automated install/basic-install.sh]: @Mcat12 I suppose we could change the filename to `pihole_install.log`, too.  But it should also be used as a variable here.  I'm sure there are lots of cases like this (hardcoding something that is already a variable).  It's a minor change, but probably still worthwhile to do as a best practice.

Comment (Mausy5043) [reply automated install/basic-install.sh]: @jacobsalmela : Changed the static text to variable.

Comment (Mausy5043) [reply automated install/basic-install.sh]: You also want to change the minus sign to an underscore? Or keep that change for later?

Comment (PromoFaux) [reply automated install/basic-install.sh]: Might as well make the change now :)

**Comment by Mausy5043** [comment]
There ya go.

**Comment by dschaper** [comment]
I have a similiar PR that I was going to push this afternoon that I can include some of these changes in. If we are going to move and put the log files in `/var/log` now and will have `debug` and `install` log files in it, I would like to put them in ` /var/log/pihole/` and timestamp the log files. `/var/log/pihole/install-MMDDYY-HHMMSS.log` and `/var/log/pihole/debug-MMDDYY-HHMMSS.log` instead of overwriting over each log file on an install. This is to help when there are multiple runs of the installer or debugger so that we can see changes between files. Also, I'd like to use the closed file handle technique instead of having a copy of the file in `/tmp` as that file should be deleted at the end of the script run and instead of adding in a deletion, it's just easier to create a closed file and access it through a file handle.

**Comment by dschaper** [comment]
The closed file accessed via handle is detailed in https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/piholeDebug.sh#L464

**Comment by dschaper** [comment]
And looking at ISO 8601, it probably should be YYMMDD-HHMMSS.log just to be in compliance with the standards for datestamps.

**Comment by dschaper** [comment]
I was planning on having this pushed for review this weekend, but I'm swamped with my real life stuff, I promise to have it up in the next day or two, I doubt I'll have it tonight, but tomorrow or the next day and I'll push a new PR since I don't have access to the branch that this PR was created on. Sorry, it will make it in 3.2.

**Comment by dschaper** [comment]
Please see #1773

**Comment by dschaper** [comment]
I am going to close this as we address it in the above PR. The hold on this is that we need to coordinate with a few things to move the log files with all the modules. (Pi-hole, debugger and FTL). That is why it did not go out in v3.2. It will be slated for a point release in the near future. I apologize for the delay but outside extraordinary circumstances pulled me away from the project for a period of time and I was unable to push this to get it done by v3.2.

**Pull request closed without merging**